src/events/crawler/lib/get.js

```diff
@@ -61,7 +62,7 @@ export const get = async (url, type, date = process.env.SCRAPE_DATE || datetime.
       }
 
       let errorMsg = '';
-      const response = await needle('get', url).catch(err => {
+      const response = await needle('get', url, { cookies }).catch(err => {
```

Comment:
I'm not a real js developer, but the hash feels like it needs a key, something like

```
opts = { cookies: cookies }
const response = await needle('get', url, opts).catch(err => {
```

because `needle(...)` actually gets the cookies by a lookup:

```
  if (options.cookies)
    config.headers['cookie'] = cookies.write(options.cookies);
```

In addition this change will cause a null/undefined cookie to be written for every call to `get` that doesn't include the cookie.  I don't know if that's problematic or not, but it feels wrong.